extkeys/mnemonic.go

```diff
@@ -53,14 +54,14 @@ const (
 
 // Languages is a list of supported languages for which mnemonic keys can be generated
 var Languages = [...]string{
-	"English",
```

Comment:
Why did this change?

Reply:
I'd changed them to lower-case to match the case of the languages specified in the seed vectors when I was running the `ValidateMnemonic` test against all of the seed vectors.  This can be changed back as needed I think.

Reply:
Do you want me to change this back?

Reply:
Are these seed vectors in this repo? Or it was a temporary change and tested with a code that is not included in this PR? I would minimize the number of changes if not necessary.